src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,29 +1,50 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone CP"
-      breadcrumbs: Zone CPs
+      title: "{name} Control Plane"
+      breadcrumbs: Control Planes
       config:
         no-subscriptions: This zone has no subscriptions
     items:
-      title: Zone CPs
-      breadcrumbs: Zone CPs
+      title: Control Planes
+      breadcrumbs: Control Planes
+  list:
+    tableHeaders:
```

Comment:
```yaml
zone-cps.route.items.headers:
  version:
  type:
  status:
  warnings:
# or (up to you)
zone-cps.route.items.collection.headers:
  version:
  type:
  status:
  warnings:
```

```yaml
common.collection.headers:
  actions: Actions
```

Alternatively, as I know this is all about rolling out DataSource so we shouldn't really be doing i18n at the same time, I'd be fine if you removed this and come back later when we can go through and i18n all the collections together in one go to prevent going over this all over again.

Lmk

Reply:
Done.

Reply:
I've unresolved this as I spotted a few places where we are still using camelCasing and not pulling `actions` from `common`

Reply:
I have no intention of going about finding all instances of camel-cased message keys. Also, the table headers are pulled from the zone message file. I won’t move them into common because they don’t belong there. Aside from that, the _actions_ actually all reference the common message file.

Resolving this again.

Reply:
Ok so it looks like you've changed some of these now, let me take another look.

> I have no intention of going about finding all instances of camel-cased message keys

I don't want you to do that, only stop adding new ones, there were plenty of new ones in this PR.

>  Also, the table headers are pulled from the zone message file. I won’t move them into common because they don’t belong there. Aside from that, the actions actually all reference the common message file.

This sounds like what I would like, let me look again now you've made these other changes.